src/lib/y2network/sysconfig/config_reader.rb

```diff
@@ -106,14 +107,14 @@ def load_routes_from(path = nil)
       #
       # return [Boolean] true when IPv4 forwarding is allowed
       def forward_ipv4?
-        Yast::SCR.Read(Yast::Path.new(SYSCTL_IPV4_PATH)) == "1"
+        sysctl_file.forward_ipv4 == "1"
       end
 
       # Reads IPv6 forwarding status
       #
       # return [Boolean] true when IPv6 forwarding is allowed
       def forward_ipv6?
-        Yast::SCR.Read(Yast::Path.new(SYSCTL_IPV6_PATH)) == "1"
+        sysctl_file.forward_ipv6 == "1"
```

Comment:
when I see this, I think that logic for understanding values should be in CFA model. So maybe add helpers in sysctl model to have `forward_ipv6?` which returns boolean and ideally also `forward_ipv6=` that understand boolean and convert it to 1/0.

Reply:
To some extent, we are representing the file content, so I wanted to avoid conversions. Adding a `#forward_ipv6?` method is OK.